# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/webrender_bindings/DCLayerTree.cpp
# Commit: 5adde51a4a33
# Full Hash: 5adde51a4a3382b0c577c4e88748d7f15865c47a
# Author: Sotaro Ikeda <sotaro.ikeda.g@gmail.com>
# Date: 2025-06-13 21:12:52
# Regressor Bug: 1945683
# File Overlap Count: 1
# Description:
#   Bug 1945683 - Enable layer compositor on Windows in Nightly r=gfx-reviewers,lsalzman
#   
#   All known rendering bugs of layer compositor have been addressed. Then is seems OK to remove nightly only limitation of pref gfx.webrender.layer-compositor.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D251788
# ==============================================================================

diff -r e0425420b975 -r 5adde51a4a33 gfx/webrender_bindings/DCLayerTree.cpp
--- a/gfx/webrender_bindings/DCLayerTree.cpp	Fri Jun 13 13:58:02 2025 +0000
+++ b/gfx/webrender_bindings/DCLayerTree.cpp	Tue Jun 10 11:15:01 2025 +0000
@@ -547,13 +547,8 @@
 }
 
 bool DCLayerTree::UseLayerCompositor() const {
-// Only allow the layer compositor in nightly builds, for now.
-#ifdef NIGHTLY_BUILD
   return UseNativeCompositor() &&
          StaticPrefs::gfx_webrender_layer_compositor_AtStartup();
-#else
-  return false;
-#endif
 }
 
 void DCLayerTree::DisableNativeCompositor() {